### PR TITLE
build fix + tagnum OutputExtend fix

### PR DESCRIPTION
**Pull request by yannicklamprecht**
- ~~This will upgrade the tooling to Gradle 5~~
- Fix building -> no errors are thrown only the deprecated libraries warning
- Adds the OutputExtend fix from TagnumElite

### CODE REVIEW

File: worldedit-bukkit/build.gradle [hunk 1]
```diff
@@ -54,27 +54,29 @@ processResources {
...
+task createPom {
+    doLast {
+        pom {
+            project {
+                groupId 'com.boydti'
+                artifactId 'fawe-bukkit'
+                version project.parent.version
+            }
         }
-    }
```
Comment (N0tMyFaultOG) [inline worldedit-bukkit/build.gradle]: Unnecessary changes

File: worldedit-core/build.gradle [hunk 2]
```diff
@@ -70,37 +70,39 @@ processResources {
     }
 }
 
-jar.archiveName="fawe-api-${project.parent.version}.jar"
```
Comment (N0tMyFaultOG) [inline worldedit-core/build.gradle]: If you want to update gradle, update it properly by updating deprecated methods too.

File: worldedit-bukkit/build.gradle (hunk 1 above)
Comment (yannicklamprecht) [reply worldedit-bukkit/build.gradle]: Yep the formatting needs to be adjusted. Will do it tomorrow.  I'm going to bed now. Checking that the checkstyle can be used for formatting. Thanks

File: worldedit-core/build.gradle [hunk 3]
```diff
@@ -70,37 +70,39 @@ processResources {
...
+jar.archiveName = "fawe-api-${project.parent.version}.jar"
 jar.destinationDir = file '../mvn/com/boydti/fawe-api/' + project.parent.version
```
Comment (N0tMyFaultOG) [inline worldedit-core/build.gradle]: Deprecated

File: worldedit-core/build.gradle (hunk 2 above)
Comment (yannicklamprecht) [reply worldedit-core/build.gradle]: I'm looking into it as well tomorrow. The rest regarding Gradle follows the same comment.

Comment (N0tMyFaultOG) [reply worldedit-core/build.gradle]: 👍 
I'll just annotate it once.

**Comment by MattBDev** [comment]
> Actually, I'd not edit the build scripts. Since we plan to use Kotlin DSL and have all scripts migrated and updated, there isn't a big need to update them on here.

We already are using Kotlin DSL since all the actual work is being done on the commanding branch.

**Comment by yannicklamprecht** [comment]
So I'll revert the Gradle stuff and get it working??

File: build.gradle [hunk 4]
```diff
@@ -141,9 +141,12 @@ configure(['worldedit-core', 'worldedit-bukkit'].collect { project("$it") }) {
         build.dependsOn(sourcesJar)
     }
 
+    compileJava.options.encoding = 'UTF-8'
+
 //    build.dependsOn(checkstyleMain)
 //    build.dependsOn(checkstyleTest)
 
+
```
Comment (yannicklamprecht) [inline build.gradle]: remove

File: build.gradle [hunk 5]
```diff
@@ -141,9 +141,12 @@ configure(['worldedit-core', 'worldedit-bukkit'].collect { project("$it") }) {
         build.dependsOn(sourcesJar)
     }
 
+    compileJava.options.encoding = 'UTF-8'
+
```
Comment (yannicklamprecht) [inline build.gradle]: remove

File: worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/PlotSetBiome.java [hunk 6]
```diff
@@ -48,53 +48,53 @@ public PlotSetBiome() {
...
         final BiomeType biome = Biomes.findBiomeByName(knownBiomes, args[0], biomeRegistry);
         if (biome == null) {
-            String biomes = StringMan.join(WorldUtil.IMP.getBiomeList(), Captions.BLOCK_LIST_SEPARATER.s());
+
+            String biomes = StringMan.join(WorldUtil.IMP.getBiomeList(), ", ");
             Captions.NEED_BIOME.send(player);
-            MainUtil.sendMessage(player, Captions.SUBCOMMAND_SET_OPTIONS_HEADER.s() + biomes);
-            return;
+            MainUtil.sendMessage(player, Captions.format(Captions.SUBCOMMAND_SET_OPTIONS_HEADER, biomes));
+
```
Comment (yannicklamprecht) [inline worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/PlotSetBiome.java]: remove

File: worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/PlotSetBiome.java [hunk 7]
```diff
@@ -48,53 +48,53 @@ public PlotSetBiome() {
...
+            return CompletableFuture.completedFuture(true);
         }
         checkTrue(args.length == 1, Captions.COMMAND_SYNTAX, getUsage());
         final HashSet<RegionWrapper> regions = plot.getRegions();
         BiomeRegistry biomeRegistry = WorldEdit.getInstance().getPlatformManager().queryCapability(Capability.GAME_HOOKS).getRegistries().getBiomeRegistry();
         Collection<BiomeType> knownBiomes = BiomeTypes.values();
         final BiomeType biome = Biomes.findBiomeByName(knownBiomes, args[0], biomeRegistry);
         if (biome == null) {
-            String biomes = StringMan.join(WorldUtil.IMP.getBiomeList(), Captions.BLOCK_LIST_SEPARATER.s());
+
```
Comment (yannicklamprecht) [inline worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/PlotSetBiome.java]: remove

File: worldedit-bukkit/build.gradle (hunk 1 above)
Comment (yannicklamprecht) [reply worldedit-bukkit/build.gradle]: reverted the Gradle changes cause Gradle upgrade is done in commanding branch.

File: worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/FaweChunkManager.java [hunk 8]
```diff
@@ -30,15 +30,14 @@ public FaweChunkManager(ChunkManager parent) {
     }
 
     @Override
-    public CompletableFuture loadChunk(String world, ChunkLoc loc, boolean force) {
```
Comment (N0tMyFaultOG) [inline worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/FaweChunkManager.java]: Was that reverted on purpose?

**Comment by yannicklamprecht** [comment]
@N0tMyFaultOG 
>Was that reverted on purpose?
Yes, I reverted the Gradle changes cause Gradle upgrade is done in commanding branch.

**Comment by N0tMyFaultOG** [comment]
Already deleted my comment, wrong pr

File: build.gradle [hunk 9]
```diff
@@ -141,6 +141,7 @@ configure(['worldedit-core', 'worldedit-bukkit'].collect { project("$it") }) {
         build.dependsOn(sourcesJar)
     }
 
+    compileJava.options.encoding = 'UTF-8'
```
Comment (MattBDev) [inline build.gradle]: I'm fine with minor change. No harm in it.

File: worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/wrapper/AsyncChunk.java [hunk 10]
```diff
@@ -169,7 +170,7 @@ public boolean unload() {
 
     @Override
     public boolean isSlimeChunk() {
-        return false;
+        return world.getChunkAt(x, z).isSlimeChunk();
```
Comment (MattBDev) [inline worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/wrapper/AsyncChunk.java]: This wasn't really needed since AsyncChunk will only be accessed by FAWE but I'm with with it still being here.

File: worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/FaweLocalBlockQueue.java [hunk 11]
```diff
@@ -114,9 +111,8 @@ public void flush() {
     }
 
     @Override
-    public void enqueue() {
-        super.enqueue();
-        IMP.enqueue();
+    public boolean enqueue() {
```
Comment (MattBDev) [inline worldedit-core/src/main/java/com/boydti/fawe/regions/general/plot/FaweLocalBlockQueue.java]: Smart change.
